repos/system_upgrade/el9toel10/actors/checkpamuserdb/libraries/checkpamuserdb.py

```diff
@@ -0,0 +1,22 @@
+from leapp import reporting
+from leapp.libraries.stdlib import api
+from leapp.models import PamUserDbLocation
+
+
+def process():
+    for msg in api.consume(PamUserDbLocation):
+        locations = msg.locations
+
+    if locations:
+        reporting.create_report([
+            reporting.Title('pam_userdb database will be converted to GDBM'),
+            reporting.Summary(
+                'pam_userdb used BerkeleyDB as the backend database format. '
+                'It is no longer maintained as open source, so it will be '
+                'replaced by GDBM. The upgrade process has detected '
+                'BerkeleyDB databases [{}] in the system and it will proceed to '
+                'convert them.'.format(locations)),
```

Comment:
@ikerexxe  is it important for people to know what will be the new filenames? there is changed extension `.db` to `.gdbm`. In case it's important, I would mention it in the summary too.

Reply:
I wouldn't say it's important, so let's skip mentioning it

Reply:
@matejmatuska the msg is good.

Reply:
Done